src/common/liboptparse/optparse.c

```diff
@@ -1216,6 +1216,8 @@ static char * optstring_append (char *optstring, struct optparse_option *o)
      *   an option with no argument (has_arg = 0), 2 characters
      *   for a required argument (has_arg = 1), and 3 chars for
      *   an optional argument "o::" (has_arg = 2)
+     *  N.B.: optional argument is applied to short options only iff
```

Comment:
Sorry, did you mean "only iff" or just "iff"?

Reply:
blah! typo. "iff" would work, but "only if" is probably more clear. Wil fix.

Reply:
Thanks for catching that, force pushed a fix.

Reply:
Thanks!  Feel free to set MWP!